lib/Command/Room/TRoomCommand.php

```diff
@@ -335,4 +304,60 @@ protected function removeRoomModerators(Room $room, array $userIds): void {
 			$room->setParticipantType($participant, Participant::USER);
 		}
 	}
+
+	protected function completeTokenValues(CompletionContext $context): array {
+		/** @var Manager $roomManager */
+		$roomManager = \OC::$server->query(Manager::class);
+		return array_map(function (Room $room) {
+			return $room->getToken();
+		}, $roomManager->searchRoomsByToken($context->getCurrentWord()));
+	}
+
+	protected function completeUserValues(CompletionContext $context): array {
+		$userManager = \OC::$server->getUserManager();
```

Comment:
Use DI in the command and inject it into this method?

Same for the other managers?

Reply:
Done :+1: :smiley: